curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/use-capture-groups-to-search-and-replace.english.md

```diff
@@ -38,10 +38,17 @@ Write a regex so that it will search for the string <code>"good"</code>. Then up
 tests:
   - text: You should use <code>.replace()</code> to search and replace.
     testString: assert(code.match(/\.replace\(.*\)/));
-  - text: Your regex should change <code>"This sandwich is good."</code> to <code>"This sandwich is okey-dokey."</code>
-    testString: assert(result == "This sandwich is okey-dokey." && replaceText === "okey-dokey");
+  - text: Your regex should change <code>"one two three"</code> to <code>"three two one"</code>
+    testString: assert(result === "three two one");
   - text: You should not change the last line.
-    testString: assert(code.match(/result\s*=\s*huhText\.replace\(.*?\)/));
+    testString: assert(code.match(/result\s*=\s*str\.replace\(.*?\)/));
+  - text: <code>fixRegex</code> should use at least three capture groups.
+    testString: assert((new RegExp(fixRegex.source + '|')).exec('').length - 1 >= 3);
+  - text: <code>replaceText</code> should use parenthesized submatch string(s).
```

Comment:
> <code>replaceText</code> should use parenthesized submatch string(s).

If I'm new to regex, going solely by the challenge description, I have no idea what "parenthesized submatch string(s)" means. Probably should be rephrased.

Reply:
Any suggestions on what to rephrase it to?  I didn't come across or previously know an alternate term. If I were a new(er) student and making use of the "Read-Search-Ask" slogan that was so prominent on FCC, a google search of the phrase would produce accurate results (for multiple programming languages). If I were me and had the habit of looking up js docs on MDN, the String.prototype.replace() page has this exact wording.

I have no strong feelings about changing it to something else, I just have no idea what would be _better_. I could use some help there.

Reply:
An/Or I suppose we could add the preferred terminology to the description. At present it says something to the effect of "use dollar signs", but offers no terminology to concisely describe that.

Reply:
Since you have now edited the instructions to add a little detail about the `$` syntax, perhaps this could be (re)used to simplify the terminology - 

> text: <code>replaceText</code> should use parenthesized submatch string(s) (ie. the nth parenthesized submatch string, `$n`, corresponds to the nth capture group).

The MDN docs on `.replace()` uses this terminology.